README.md

```diff
@@ -89,3 +89,24 @@ After cloning this repository, these are the 2 steps you should follow:
 2. Run `./run.sh`: this will run Jenkins with the Zephyr Scale plugin installed
 3. Browse to http://localhost:8080/jenkins to access Jenkins
 
+## Configuring the Zephyr Scale Jenkins plugin using Jenkins Configuration as Code (JCasC) plugin
+You can configure the plugin by following the steps below:
+1. Install the JCasC plugin by visiting the manage plugin page and installing the Configuration as Code plugin
+2. Configure you jira Instances by creating a yaml file with the following configuration
+```yaml
+unclassified:
+  zephyr-scale:
+    jiraInstances:
+      - one:
+        value: "server"
+        serverAddress: "yourJiraServerAddress.com"
+        username: "yourJiraUsername"
+        password: "yourJiraServerPassword"
+      - two:
+        value: "cloud"
+        serverAddress: "yourJiraCloud.attlasian.net"
+        jwt: "yourCloudToken"
+        
```

Comment:
I think I'd also add somewhere something saying something along the following lines:
1. for Server configuration, the user needs to add the `value` "**server**" and the `serverAddress`, `username`, and `password`
2. for Cloud configuration, the user needs to add the `value` "**cloud**" and the `serverAddress` and `jwt`. The latter needs to be filled out with the token provided [here](https://support.smartbear.com/zephyr-scale-cloud/docs/rest-api/generating-api-access-tokens.html) 

The examples are already super clear in my opinion, but some people may not understand it fully, I think